README.md

```diff
@@ -28,7 +34,17 @@ This will start the development server on `http://localhost:8080/en/`. This shou
 
 If you want to submit a new feature or a bugfix, the best way is to create the changes in a separate branch, e.g.: `git checkout -b feature/mycoolfeature`. This will make it easier for you to submit a pull request and get your feature merged.
 
-### Layout
+### To translate content into other languages
+
+If you want to contribute in the translation to other languages or improve the current one, it isn't necessary that you work from Github, you can and should do it through Growdin, this is de correct workflow.
+
+Crowdin is an online tool that facilitates the user experience for the translator, here is more information:
+
+Website translations are handled via [Crowdin](https://crowdin.com/project/nodejs-website).
+
+To help with localization, please read the [TRANSLATION](https://github.com/nodejs/nodejs.org/blob/master/TRANSLATION.md) guide.
```

Comment:
Minor, but this can just be a local link
```suggestion
To help with localization, please read the [TRANSLATION](TRANSLATION.md) guide.
```

Reply:
Cool! A relative reference is better in this case.